app/renderer/components/main/siteInfo.js

```diff
@@ -68,8 +68,7 @@ class SiteInfo extends React.Component {
         <span className={cx({
           fa: true,
           'fa-lock': true,
-          [css(styles.secureIcon__fa)]: true,
-          [css(styles.secureIcon__extendedValidation)]: this.props.isExtendedValidation
+          [css(styles.secureIcon__fa, this.props.isExtendedValidation && styles.secureIcon__extendedValidation)]: true
```

Comment:
This doesn't follow the BEM strictly; I'll work on the file including this line with another PR.